presets/advertising_indication.yml

```diff
@@ -0,0 +1,114 @@
+--- 
+author: PaDe
+description: 
+  default: "Pour participer à l'inventaire des préenseignes publicitaires en France, qui sont presque à 100% illégales."
+  en: "To participate in inventorying advertising indication boards in France, which are almost 100% unlawfull."
```

Comment:
Si possible, merci de rester objectif dans la description, sans jugement de valeur ("qui sont presque 100% illégales")

Proposition: remplacer la description anglaise par celle-ci:
"To participate in the creation of the inventory for advertising indicator boards"